.markdown-link-check.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "aliveStatusCodes": [200, 206, 402],
```

Comment:
Is it possible to do ranges? We prob want all 200's, all 300's, etc

Reply:
Based on my brief chat-gpt research it's not easily doable programatically. I just copied this from the main langgraph one so I assume someone smarter than me had a good reason for selecting these codes but idk